src/impedancefitter/time_domain/signals.py

```diff
@@ -31,3 +31,54 @@ def rectangle(
             if biphasic:
                 signal[idx] = -amplitude
     return signal
+
+
+def biphasic_with_time_shift(
```

Comment:
Could you please add a docstring here to document the function?

Reply:
@CheLamVien ping, please only add a simple docstring to fix the pre-commit ci pipeline